lib/client.js

```diff
@@ -104,16 +126,24 @@ function messageHandler(client, event) {
   }
 }
 
-var Client = function(origin, appGuid) {
-  this._origin = origin;
+var Client = function(options) {
+  this._parent = options.parent;
```

Comment:
I'd think I'd prefer `root` to `parent`, which seems loaded with meaning in iframe context